extensions/wikia/CuratedContent/js/CuratedContentToolButton.js

```diff
@@ -0,0 +1,15 @@
+require(
+	[
+		'jquery'
+	],
+	function ($) {
+		$('#CuratedContentTool').click(function () {
+			var iframe = '<iframe data-url="/main/edit?useskin=mercury" id="CuratedContentToolIframe" name="curated-content-tool" width="100%" height="500"></iframe>',
+				title = 'Mobile Main Page';
+
+			require(['curatedContentTool.modal'], function (curatedContentToolModal) {
```

Comment:
are you sure about not loading the file everywhere? I thought curated_content_tool_button_js  asset's manager group is loaded when this is called https://github.com/Wikia/app/pull/8153/files#diff-f2c81fb514ec8b3f2c72e17d647ac290R50